apprise/plugins/NotifyGitter.py

```diff
@@ -99,7 +100,7 @@ class NotifyGitter(NotifyBase):
     request_rate_per_sec = 0
 
     # For Tracking Purposes
-    ratelimit_reset = datetime.utcnow()
+    ratelimit_reset = datetime.now(timezone.utc).replace(tzinfo=None)
```

Comment:
These `ratelimit_reset` related changes here and elsewhere aren't that nice, but they keep the type of the field as naïve datetime; changing a user visible field to aware could be seen as a breaking change.